react/components/molecules/snackbar-ios/snackbar-ios.js

```diff
@@ -0,0 +1,126 @@
+import React, { PureComponent } from "react";
+import { Animated, StyleSheet, ViewPropTypes, Platform, Text } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { baseStyles, IdentifiableMixin } from "../../../util";
+
+import { Touchable } from "../touchable";
+
+export class SnackbarIos extends mix(PureComponent).width(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            text: PropTypes.string,
+            actionText: PropTypes.string,
+            duration: PropTypes.number,
+            animationDuration: PropTypes.number,
+            onActionPress: PropTypes.func,
+            style: ViewPropTypes.style
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            text: undefined,
+            actionText: undefined,
+            duration: 5000,
+            animationDuration: 300,
+            style: {}
+        };
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            text: this.props.text,
+            actionText: this.props.actionText,
+            animationDuration: this.props.animationDuration,

```

Comment:
remove from state since they are not being used

Reply:
I know you took this from ToastMessage, but that maybe also remove there, since it's not needed